### ISSUE TITLE
Exercise insufficient

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10246]
**Preferred term label:**
Exercise insufficient

**Synonyms**
Lack of physical exercise

**Definition (free text, please give PubMed ID)**
The opposite of https://hpo.jax.org/app/browse/term/HP:0025377 but not the same as https://hpo.jax.org/app/browse/term/HP:0003546

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Insufficient exercise syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
The triggered by terms are supposed to represent things that we observe to trigger, but here it is probably best to put the term into the PMH section.
I have added the following, please reopen if needed

Adding new term: 
- ID: HP:6000023
- label: Lack of sufficient daily physical activity
- parents: HP:0032443
- pmid: PMID:23798298
- def: Physical activity levels less than those required for optimal health and prevention of premature death.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10246
- synonyms: physical inactivity(NONE)
